node/index.d.ts

```diff
@@ -1891,8 +1891,27 @@ declare module "dns" {
         priority: number
     }
 
-    export function lookup(domain: string, family: number, callback: (err: Error, address: string, family: number) => void): string;
-    export function lookup(domain: string, callback: (err: Error, address: string, family: number) => void): string;
+    // Supported getaddrinfo flags
+    export const ADDRCONFIG: number;
+    export const V4MAPPED: number;
+
+    export interface LookupOptions {
+        family?: number;
+        hints?: number;
+        all?: boolean;
+    }
+
+    interface Address {
+        address: string;
+        family: number;
+    }
+
+    interface QueryReqWrap {}
```

Comment:
I think this should be GetAddrInfoReqWrap.
Not sure if we should add return value of dns.lookup here as this is not documented by Node. Also the type GetAddrInfoReqWrap is not documented so in the end it's of no use and to me it looks more like an internal detail.